### PR TITLE
Bakursait lp renaming variables

### LINKED ISSUE #104: Embedded Documentation - Add units to variable names
- Variables for code we have written need to have reference units, such as referring to how large the world is, what is the world actually measured in.

- Add Literate Programming to the code.

Done: When code we have written have units in the variables to define what the number is and the units it is measured in.

### PR DESCRIPTION
**Pull request by bakursait2**
[Issue#104](https://github.com/WSUCEG-7140/Tempus_Fugit_Minecraft/issues/104)
- Added Literate Programming.
- Fix variables naming.

### CODE REVIEW

**Review by xieWright — APPROVED** [review]
Looked it over, we got the examples of literate programming and programming by contract it looks. Also our names and gitIdDs. Nothing was broken, so it passes the smell test.

**Review by criticalglitch — APPROVED** [review]
Good names for variables.

File: tests/test_world.py [hunk 1]
```diff
@@ -103,13 +103,14 @@ def test_generate_single_tree_custom_values(self):
     #issue84
     def test_generate_single_cloud_blocks_at_a_specific_height(self):
         y = 100
-        cloud = World.generate_single_cloud(cloud_center_x=0,cloud_center_y=y,cloud_center_z=0,s=3)
+        cloud = World.generate_single_cloud(cloud_center_x_coordinate_in_model=0,cloud_center_y_coordinate_in_model=y,cloud_center_z_coordinate_in_model=0,\
+            blocks_from_center_of_clouds_to_end_in_one_direction=3)
```
Comment (w1158mxs) [inline tests/test_world.py:107]: :) longest variable I have ever seen! but so specific, good one!

**Review by w1158mxs — APPROVED** [review]
Thank you

**Pull request merged**